src/client/app/components/pages/RootPage/index.tsx

```diff
@@ -31,12 +34,13 @@ import theme from '../../../theme'
 
 type RootProps = {
   store: Store
+  history: History
 }
 
-const Root: FunctionComponent<RootProps> = ({ store }: RootProps) => (
+const Root: FunctionComponent<RootProps> = ({ store, history }: RootProps) => (
   <Provider store={store}>
     <MuiThemeProvider theme={theme}>
-      <HashRouter>
+      <Router history={history}>
```

Comment:
are you confident that this change doesn't break anything? what does the new route look like? will it clash with the redirection endpoint?

Reply:
New route looks exactly the same - using `<HashRouter>` and using `<Router>` with `createHashHistory()` are equivalent.

Reply:
thanks!